patches/minecraft/net/minecraft/world/entity/npc/VillagerType.java.patch

```diff
@@ -0,0 +1,15 @@
+--- a/net/minecraft/world/entity/npc/VillagerType.java
++++ b/net/minecraft/world/entity/npc/VillagerType.java
+@@ -69,4 +_,12 @@
+     public static VillagerType byBiome(Holder<Biome> p_204074_) {
+         return p_204074_.unwrapKey().map(BY_BIOME::get).orElse(PLAINS);
+     }
++
++    /** FORGE: Registers the VillagerType that will spawn in the given biome.
++     * @apiNote This method should be called from the {@link net.minecraftforge.fml.event.lifecycle.FMLCommonSetupEvent} event using one of the {@link
++     * net.minecraftforge.fml.event.lifecycle.ParallelDispatchEvent} enqueueWork methods.
++     **/
```

Comment:
No need to throw in an API Note tag. In the same line as the comment is fine without the link.

Reply:
Remove the @apiNote and the @link?

Reply:
At the very least keep everything in one line.